specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 2095
```
  ballots_under_audit_table: TYPE =
    sequence[[ballot, ballot_manifest_info, list[ballot_mark],
              cvr, margin_overstatement, margin_understatement]]
```
It's not clear to me how the `ballot` helps here.
It seems that, at least on this go-round, we won't get a list of ballot_marks, but rather in effect a CVR generated by the auditors, which CORLA2 has called an aCVR (audit CVR).
If you accept my suggestion that cvr include ballot_manifest_info, it wouldn't be necessary here.

Having data on when ballot interpretations are added is critical for the sort of efficiency / time-and-motion studies that we'll want to do to learn from this unique experience.  If that data isn't available in some other place, it would be great to add a timestamp for when the audit interpretation was first entered.
Those timestamps should be included in the the data dump that is made available to the public.
Ideally (someday) we would add another one for when it was last modified, if it can be modified.